app/ui/src/app/integration/edit-page/current-flow.service.ts

```diff
@@ -90,6 +90,14 @@ export class CurrentFlowService {
     this._integration = undefined;
   }
 
+  isSaved(): boolean {
+    /**
+     * TODO: check if this is the proper way to check if the integration has been
```

Comment:
So far yes, this is all we use.

We *could* also use a metadata item to flag this, as well add a dirty flag actually which could be handy in the future, then it'd be in the `integration.metadata` map.  I can't recall if it allows values other than strings so that could be only one gotcha.

Reply:
BTW I'm not suggesting you do it for this PR :-)